### PR TITLE
[nnpackage] Add mx dtypes to schema

### PR DESCRIPTION
**Pull request by jinevening**
This adds mxfp4 and mxint8 to circle schema.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to: https://github.com/Samsung/ONE/issues/14293
Draft PR: #14294

### CODE REVIEW

File: nnpackage/schema/circle_schema.fbs [hunk 1]
```diff
@@ -35,6 +35,7 @@
 // Version 0.8: GRU op is added. UINT4 is added.
 // Version 0.9: GGML_Q{X}_{Y} types are added. Weight compression option is added.
 //              ROPE op is added.
+// Version 0.10: MXFP4, MXINT8 types are added.
```
Comment (seanshpark) [inline nnpackage/schema/circle_schema.fbs:38]: from previous update PR comment, https://github.com/Samsung/ONE/pull/14013#issuecomment-2370118318 , I think we have to stick to 0.9

**Review by glistening — APPROVED** [review]
LGTM. I also thought 0.10 is right, also prefer 0.10. However, other members especially who publishe something using circle schema seems ok using 0.9, if there is not public release in ONE.

File: nnpackage/schema/circle_schema.fbs [hunk 2]
```diff
@@ -78,6 +78,11 @@ enum TensorType : byte {
   GGML_Q4_1 = -3,
   GGML_Q8_0 = -4,
   GGML_Q8_1 = -5,
+
+
+  // MX dtypes
```
Comment (seanshpark) [inline nnpackage/schema/circle_schema.fbs:83]: if there is no particular reason for two line spaces...
```suggestion

  // MX dtypes
```

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

**Comment by jinevening** [comment]
I'll merge this PR as @glistening already approved (https://github.com/Samsung/ONE/pull/14299#pullrequestreview-2414478480).

**Pull request merged**